README.md

```diff
@@ -126,20 +126,23 @@ The commands will do the following:
 </details>
 <br>
 
-Connect VS Code to your JFrog Platform instance by clicking on the green Connect ![Connect](resources/readme/connect.png) button or the provided button in the JFrog extension tab:
-![Connect](resources/readme/preview/connect.png)
+Once you installed, You can view the sigh in page by clicking on the Frog extension tab:
```

Comment:
Once you installed, You can view the sigh in page by clicking on the Frog extension tab: LoginPage
-->
Once the JFrog Extension is installed in VS Code, click on the JFrog tab.

(Let's add an image of the JFrog tab here)